examples/dhf5-custom-hook/build.gradle

```diff
@@ -0,0 +1,47 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+        mavenLocal()
+    }
+    dependencies {
+        classpath("com.marklogic:ml-data-hub:${mlDHFVersion}")
+    }
+}
+plugins {
+    id 'net.saliman.properties' version '1.4.6'
+}
+
+apply plugin: 'com.marklogic.ml-data-hub'
+
+repositories {
+    jcenter()
+    maven { url "http://developer.marklogic.com/maven2/" }
```

Comment:
we should probably remove the internal, svn only accessible repo from public consumption

Reply:
That one is public - though I don't think it's needed, I'll check it out. 

Also going to do some performance testing of this, so don't merge this quite yet. Want to see if there's any noticeable impact with passing a bunch of context to the custom hook.